airbyte-cdk/bulk/core/load/src/test/resources/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+---
+data:
+  dockerRepository: "airbyte/fake-source"
```

Comment:
I'll update this file, it's needed to make something in the connector runner happy